TypeCobol/Compiler/Sql/CodeElements/SqlCodeElementCheckers.cs

```diff
@@ -7,19 +8,38 @@ namespace TypeCobol.Compiler.Sql.CodeElements
 {
     public static class SavepointStatementChecker
     {
-        public static void OnCodeElement(SavepointStatement savepointStatement,
-            CodeElementsParser.SavepointStatementContext context)
+        public static void OnCodeElement(SavepointStatement savepointStatement, CodeElementsParser.SavepointStatementContext context)
         {
             if (savepointStatement.Name != null)
             {
                 var savePointName = savepointStatement.Name.ToString();
                 if (savePointName.StartsWith("SYS", StringComparison.OrdinalIgnoreCase))
                 {
                     DiagnosticUtils.AddError(savepointStatement,
-                        "Invalid savepoint-name, it must not begin with 'SYS'.",
-                        context);
+                        "Invalid savepoint-name, it must not begin with 'SYS'.", context);
                 }
             }
         }
     }
+    public static class IntegrateAlterSequenceStatementChecker
+    {
+        public static void OnCodeElement(AlterSequenceStatement alterSequenceStatement, List<SqlCodeElementBuilder.AlterSequenceClauseTypes> duplicatedClauses,
+            CodeElementsParser.AlterSequenceStatementContext context)
+        {
+            foreach (var clauseType in duplicatedClauses)
+            {
+                DiagnosticUtils.AddError(alterSequenceStatement, clauseType + " can not be specified more than once",
+                    context);
+            }
+
+            if (alterSequenceStatement != null && alterSequenceStatement.MinValue == null &&
+                alterSequenceStatement.MaxValue == null &&
+                alterSequenceStatement.Ordered == null && alterSequenceStatement.Restart == null &&
+                alterSequenceStatement.IncrementValue == null && alterSequenceStatement.Cycle == null &&
+                alterSequenceStatement.NoCache == false)
```

Comment:
Better test on CacheValue, if we decide to distinguish between explicit NO CACHE and implicit